src/projections/igh.cpp

```diff
@@ -186,14 +186,17 @@ static PJ *destructor (PJ *P, int errlev) {
     -180    -100      -20         80          180
 */
 
-#define SETUP(n, proj, x_0, y_0, lon_0) \
-    if (!(Q->pj[n-1] = pj_##proj(nullptr))) return destructor(P, ENOMEM); \
-    if (!(Q->pj[n-1] = pj_##proj(Q->pj[n-1]))) return destructor(P, ENOMEM); \
-    Q->pj[n-1]->ctx = P->ctx; \
-    Q->pj[n-1]->x0 = x_0; \
-    Q->pj[n-1]->y0 = y_0; \
+static bool setup_zone(PJ *P, struct pj_opaque *Q, int n, \
```

Comment:
no need for those trailing '\'.

Reply:
I thought they added a personal touch!  😄 

(Just pushed a commit removing them.)